.huskyrc.json

```diff
@@ -1,5 +1,5 @@
 {
-    "hooks": {
-        "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
-    }
+  "hooks": {
+    "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
```

Comment:
this is an older way of doing husky commit validation which forces us to pin the husky version in package.json to an old version 

new way is detailed here
https://typicode.github.io/husky/#/?id=husky_git_params-ie-commitlint-

Reply:
Great callout, I forgot to remove this

Reply:
never mind i see you added the commit-msg file below